examples/presigned_post_policy.py

```diff
@@ -33,10 +33,21 @@
 
 form_data = client.presigned_post_policy(policy)
 
+# Decode from bytes so POST will work by having the policy as policy=eyJleHBpcm
+# If policy is like "policy=b'eyJleHBpcm..." Then this will fail.
+if isinstance(form_data['policy'], (bytes, bytearray)):
+    form_data['policy'] = form_data['policy'].decode("utf-8")
+
+# Create the file for the example
+# 2 MB Object to be in the defined range above in the policy.
+with open("output.txt", "wb") as out:
+    out.truncate(1 * 1024 * 1024)
```

Comment:
As we are not executing `curl` command, this fie creation is not required.

Reply:
Yeah it can be removed to keep example generic, will do

Reply:
Removed.